test/test_type.jl

```diff
@@ -686,6 +686,75 @@ isdir(outdir) && rm(outdir, recursive = true)
         end
     end
 
+    @timed_testset "Compressible Navier Stokes Diffusion 1D" begin
+        for RealT in (Float32, Float64)
+            equations = @inferred CompressibleEulerEquations1D(RealT(1.4))
+            prandtl_number() = RealT(0.72)
+            mu() = RealT(0.01)
+            equations_parabolic_primitive = @inferred CompressibleNavierStokesDiffusion1D(equations,
+                                                                                          mu = mu(),
+                                                                                          Prandtl = prandtl_number(),
+                                                                                          gradient_variables = GradientVariablesPrimitive())
+            equations_parabolic_entropy = @inferred CompressibleNavierStokesDiffusion1D(equations,
+                                                                                        mu = mu(),
+                                                                                        Prandtl = prandtl_number(),
+                                                                                        gradient_variables = GradientVariablesEntropy())
```

Comment:
Since we don't need the values of the parameters very often, I would just use normal variables here. In the examples, we also need the values in initial conditions etc, so we use functions there.

```suggestion
            prandtl_number = RealT(0.72)
            mu = RealT(0.01)
            equations_parabolic_primitive = @inferred CompressibleNavierStokesDiffusion1D(equations,
                                                                                          mu = mu,
                                                                                          Prandtl = prandtl_number,
                                                                                          gradient_variables = GradientVariablesPrimitive())
            equations_parabolic_entropy = @inferred CompressibleNavierStokesDiffusion1D(equations,
                                                                                        mu = mu,
                                                                                        Prandtl = prandtl_number,
                                                                                        gradient_variables = GradientVariablesEntropy())
```